dash/llmq/llmq_test.go

```diff
@@ -0,0 +1,60 @@
+package llmq
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/crypto"
+	"github.com/tendermint/tendermint/crypto/bls12381"
+)
+
+func Test100Members(t *testing.T) {
+	testCases := []struct {
+		name      string
+		n         int
+		threshold int
+		omit      int
+	}{
+		{
+			name:      "test 100 members at threshold",
+			n:         100,
+			threshold: 67,
```

Comment:
We might want to test for all supported llmq types, see https://github.com/dashevo/tenderdash/blob/ee269bd6899e94511647a7bc4fcb4baba72d9248/types/validator_set.go#L548